### DISCUSSION TITLE
nested with mousewheel issue confirmation? how to make it work?

### DISCUSSION THREAD

**Discussion by ysk3a** [nolimits4web/swiper#6365] [Q&A]
I'm not sure if this is only my end but I don't think issue is unique.
I have a nested swiper and used mousewheel but scrolling moves horizontally but when a nested swiper is present it still swipes horizontally instead of the swiping the nested swiper-container.
e.g. 

- https://stackblitz.com/edit/web-platform-bzo4sm?file=index.html
- https://codesandbox.io/s/happy-bose-jsgzfl?file=%2Findex.html
(same code just different online editor)
The second swiper-slide is a vertical swiper with 5 vertical slides but scrolling just moves horizontally to horizontal slide 3.

There seems to be similar issues in the issues tab of this repo but none seemed to be answered.
How can I use the mousewheel to swipe the slides of a nested swiper? maybe some kind of focus detection or metakey like shift scroll to change scroll direction?

This one is a deeply nested version I tried where the second slide is a vertical nested swiper and inside that swiper-container is another swiper-container but the issue mentioned above prevents the scrolling to slide in the correct direction.
https://codesandbox.io/s/billowing-darkness-g38slm?file=%2Findex.html

Also, nested swiper with same direction has overlapping pagination so users won't even know that there is a swiper present.

Of course the deep nested version may not be a good idea but the main problem of the scrolling of nested swiper is still present.

**Comment by hrvstr** [comment]
Hey @ysk3a, did you ever find a solution to this problem? I am facing the same issue at the moment.

**Reply by ysk3a** [reply]
Sorry forgot I made a discussion post for this library.
Did you?
I, unfortunately, did not find a solution and moved on. did not end up using the library and I just went with alternative solutions for my ui idea.
